lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -175,6 +175,14 @@
      - Indicate desired state of the instance
     choices: ['present', 'started', 'absent', 'stopped', 'restarted']
     default: present
+  unprivileged:
+    description:
+      - Indicate if the container should be privileged
+    choices: [0, 1]
+    default: 0
+    required: false
+    type: integer
+
```

Comment:
As this option has been added since the module was released, please add
`version_added: "2.3"` under `unprivileged:`